lib/broccoli/fastboot-config.js

```diff
@@ -42,14 +62,35 @@ FastBootConfig.prototype.buildDependencies = function() {
   this.moduleWhitelist = uniq(moduleWhitelist);
 };
 
+FastBootConfig.prototype.readAssetManifest = function() {
+  var assetMapPath = path.join(this.inputPaths[0], this.assetMapPath);
+
+  try {
+    var assetMap = JSON.parse(fs.readFileSync(assetMapPath));
+    return assetMap;
+  } catch (e) {
+    console.log(e.stack);
+  }
+};
+
 FastBootConfig.prototype.buildManifest = function() {
   var outputPaths = this.outputPaths;
 
-  this.manifest = {
+  var manifest = {
     appFile: strip(outputPaths.app.js),
     htmlFile: strip(outputPaths.app.html),
     vendorFile: strip(outputPaths.vendor.js)
   };
+
+  var rewrittenAssets = this.readAssetManifest();
+
+  if (rewrittenAssets) {
+    ['appFile', 'vendorFile'].forEach(function(file) {
+      manifest[file] = rewrittenAssets.assets[manifest[file]];
```

Comment:
This needs to take into account the `prepend` that was provided to broccoli-asset-rev.  

If a prepend is present, it will be included in `rewrittenAssets.assets[manifest[file]]`.